internal/http/http.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2020 Red Hat, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package http
+
+import (
+	"net"
+	"net/http"
+	"strconv"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/rhobs/kube-events-exporter/internal/collectors"
+
+	"k8s.io/klog"
+)
+
+const (
+	metricsPath = "/metrics"
+)
+
+// ServeExporterMetrics registers collectors and start serving exporter self
+// metrics on metricsPath.
+func ServeExporterMetrics(host string, port int) {
+	registry := prometheus.NewPedanticRegistry()
```

Comment:
It is not useful in the current implementation since we are not using any custom collectors. However, I thought that it might be useful in the future when we will be adding custom collectors like the one providing insight on Event's metric collections.

Reply:
Would be using it here in this registry, we are only going to be using the standard collectors? I have nothing against it, just curious.

Reply:
Indeed, I was over-engineering the Event's metric collections part.